nncf/compression_method_api.py

```diff
@@ -329,6 +329,12 @@ def _nncf_module_types_to_compress(self) -> List[str]:
                 filtered_nncf_module_names_list.append(module.__name__)
         return filtered_nncf_module_names_list
 
+    def are_frozen_layers_allowed(self) -> bool:
```

Comment:
Will there be a real use case where a user would need to query this in runtime (i.e. during training)? If not, consider making this a protected method.